hack/lib/util.sh

```diff
@@ -542,12 +542,14 @@ kube::util::download_file() {
 #  CFSSLJSON_BIN: The path of the installed cfssljson binary
 function kube::util::test_cfssl_installed {
     if ! command -v cfssl &>/dev/null || ! command -v cfssljson &>/dev/null; then
-      echo "Failed to successfully run 'cfssl', please verify that cfssl and cfssljson are in \$PATH."
-      echo "Hint: export PATH=\$PATH:\$GOPATH/bin; go get -u github.com/cloudflare/cfssl/cmd/..."
-      exit 1
```

Comment:
As this is a test function, is there a fail condition? what if download-cfssl doesn't work?